### DISCUSSION TITLE
Annatation page - How to display an annotation in the text diffrently

### DISCUSSION THREAD

**Discussion by noraojensa** [inception-project/inception#3514] [General]
Hi, we have a school project where we tackle the issue #1250 with showing a short name instead of a name that would be taking upp too much space in the text itself when creating an annotation. We have made the property and we can give the tag a short name, but have a hard time with finding the class/method that decides which name to display. We have understood that getName() is used for this purpose but can't seem to find where it is used for the tag in the featureListPanel. 

![Ann](https://user-images.githubusercontent.com/43003991/199272811-1cada0da-7bec-44b1-b742-891b6ea816b8.png)
el. 

What is the value dropdown menu called? Where can we make sure the short name is swapped out for name in the text?

**Comment by reckart** [comment]
I think that may be more tricky than expected for an issue marked "good first issue".

For a start, you could go to the project settings and configure your string feature to use the "Radio group (small tagsets)" editor type:

![Screenshot 2022-11-01 at 18 21 26](https://user-images.githubusercontent.com/1410238/199296646-4f657503-8530-4cbb-b5ae-45a964414aeb.png)

Then you can have a look at `RadioGroupStringFeatureEditor` and see if you can adjust that code to use the new short name field you have added for the tags.

For the other editors - in particular editors where the user can enter a custom value which is not in the tagset, the situation will be a bit trickier. If a user enters a value that is not in the tagset, what happens? 

Also, what happens if two tags have the same short description? Do you catch that situation already?
